2022_05_16/epitmenyado.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+import numpy as np
+
+
+def beolvasas():
+    df  = pd.read_csv("utca.txt", sep = " ", skiprows = 1, names = ["id", "s_name", "h_number", "tax_class", "area"])
+    return df
+
+def masodik_feladat(df):
+    print(f"A mintában {df.shape[0]} telek szerepel")
+
+def harmadik_feladat(df):
+    inp_adoszam = 79906 #int(input("Irjon be egy adószámot"))
+    for index, adoszam in enumerate(df.id):
+        if adoszam == inp_adoszam:
+            print(f"Egy tulajdonos adószáma: {adoszam} {df.s_name[index]} {df.h_number[index]}")
+            break
+        print("Nem szerepel az adatállományban.")
+
+def ado(df, adosav, alapterulet, epulet):
```

Comment:
Ehhez a függvényhez is írj legyszi egy kicsi docstringet, és typehintet (a typehint amit a fenti példában is csináltam, hogy a függvény arg listánál megadod, hogy mi lesz az adott argumentum típusa)